src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagTippecanoeProcessor.java

```diff
@@ -64,7 +65,7 @@ public void process(final org.openstreetmap.atlas.event.ShutdownEvent event)
     @Override
     protected String getFilename()
     {
-        return String.format("%s-%s%s", new Date().getTime(), getCount(),
-                new GeoJsonPathFilter(doesCompressOutput()).getExtension());
+        return String.format("%sP%s-%s%s", new Date().getTime(), TaskContext.getPartitionId(),
```

Comment:
what is an example of an output file name with the new partition string?

Reply:
Thanks for checking this. It made me realize I had forgotten to change this for the geojson output. 

flag: 1612311851465P5-52.log.gz
geojson: SelfIntersectingPolylineCheck-1612989473907P2-87.geojson.gz
tippecannoe: 1612311851558P0-52.geojson.gz